lib/loader.js

```diff
@@ -22,11 +23,17 @@ module.exports = function (content, sourceMap) {
     const tags = new Set()
     const file = this.fs.readFileSync(this.resourcePath).toString('utf8')
     const component = compiler.parseComponent(file)
-    component.template && compiler.compile(component.template.content, {
-      modules: [{
-        postTransformNode: node => { tags.add(node.tag) }
-      }]
-    })
+    if (component.template) {
+      // Temp fix for pug
+      if (component.template.lang === 'pug') {
+        component.template.content = pug.compile(component.template.content)()
```

Comment:
Move pug to `peerDependencies` and only require it in this block.

Reply:
Done